test/Microsoft.ML.AutoML.Tests/TestData/DatasetWithNewlineBetweenQuotes.txt

```diff
@@ -0,0 +1,36 @@
+﻿id,Column1,Column2,Column3
+1,this is a description, 1,2
+2,"this is a quote description",1,2
+3,"this is a quote description with double quote("")",1,2
+4,"this is a quote description with ""a pair of double quote""",1,2
+5,"this is a quote description with new line
+quote",1,2
+6,"this is a quote description with
+new line1 and
+new line2 and empty line
+
+and double quote""",1,2
+7, this is a description with single quote("),1,2
+// empty line between quotes
+8,"",1,2
+// single quote between quotes
+9,"""",1,2
+// simply newline between quotes
+10,"
+
+
+
+",1,2
+// simply signle quote and newline between quotes 
+11,"
+
+""""
+
+""
+
+""
+
+",1,2
+
+
+
```

Comment:
There a a variety of other places that the multi-line needs to be added.

Basically anywhere "sparse" is within the AutoML/CodeGen:
* https://github.com/dotnet/machinelearning/search?q=sparse+path%3Asrc%2FMicrosoft.ML.AutoML&unscoped_q=sparse+path%3Asrc%2FMicrosoft.ML.AutoML
* https://github.com/dotnet/machinelearning/search?q=sparse+path%3Asrc%2FMicrosoft.ML.CodeGenerator&unscoped_q=sparse+path%3Asrc%2FMicrosoft.ML.CodeGenerator

Reply:
### Resolved

Reply:
There's still some missing, like:
https://github.com/dotnet/machinelearning/blob/c0252704b3b54ebff20c5333fb449ba38ffaed92/src/Microsoft.ML.AutoML/ColumnInference/ColumnInferenceApi.cs#L31-L32

Reply:
I prefer not exposing readMultiline option in  these APIs, Simply sweeping it in Infersplit  should be enough